doc/api/buffer.markdown

```diff
@@ -248,6 +366,93 @@ console.log(buf2.toString());
   // prints: this is a tést
 ```
 
+### Class Method: Buffer.alloc(size[, fill[, encoding]])
+
+* `size` Number
+* `fill` Optional, Default `undefined`
+* `encoding` String, Optional, Default `utf8`
+
+Allocates a new `Buffer` of `size` bytes. If `fill` is `undefined`, the
+`Buffer` will be *zero-filled*.
+
+```js
+const buf = Buffer.alloc(5);
+console.log(buf);
+  // <Buffer 00 00 00 00 00>
+```
+
+The `size` must be less than or equal to the value of
+`require('buffer').kMaxLength` (on 64-bit architectures, `kMaxLength` is
+`(2^31)-1`). Otherwise, a [`RangeError`][] is thrown. If a `size` less than 0
+is specified, a zero-length `Buffer` will be created.
+
+If `fill` is specified, the allocated `Buffer` will be initialized by calling
+`buf.fill(fill)`. See [`buf.fill()`][] for more information.
+
+```js
+const buf = Buffer.alloc(5, 'a');
+console.log(buf);
+  // <Buffer 61 61 61 61 61>
+```
+
+If both `fill` and `encoding` are specified, the allocated `Buffer` will be
+initialized by calling 
+`buf.fill(Buffer.from(fill, encoding).toString('binary'))`.
```

Comment:
this long code segment may wrap and look funny. maybe just pop it to a one-liner for cleanliness.